sharding/collator/collator.go

```diff
@@ -85,11 +86,11 @@ func checkSMCForCollator(c client.Client, head *types.Header) error {
 func isAccountInCollatorPool(c client.Client) (bool, error) {
 	account := c.Account()
 	// Checks if our deposit has gone through according to the SMC
-	b, err := c.SMCCaller().IsCollatorDeposited(&bind.CallOpts{}, account.Address)
-	if !b && err != nil {
+	b, err := c.SMCCaller().NotaryRegistry(&bind.CallOpts{}, account.Address)
```

Comment:
Again, this is checking notary registry. Are there no more collators?

Reply:
No. There's no more collator. Collator has became Notary